file_feature2.two.txt

```diff
@@ -0,0 +1 @@
+second file in feature2
```

Comment:
really good you decided to add this line of code

Reply:
Thanks nabilstudies for this comment